doc/csharp/server_reflection.md

```diff
@@ -28,6 +28,24 @@ server = new Server()
 };
 server.Start();
 ```
+### Server Reflection in C# in .NET Core 3.0 sevices
+Since now a days in .NET Core 3.0, sevices are registered in startup.cs and build in program.cs, below is the way one can implement Server Reflection
```

Comment:
As I said before,  I think it would be better to just add a short note that server reflection is also supported by grpc-dotnet and just linking to the reflection example in the grpc/grpc-dotnet repository (rather than duplicating the info here).
https://github.com/grpc/grpc-dotnet/tree/master/examples#reflector